py_bind/optv/parameters.pyx

```diff
@@ -0,0 +1,58 @@
+#Implementation of Python binding to parameters.h
+from libc.stdlib cimport malloc, free
+
+cdef class Py_mm_np:
+      
+    def __init__(self, **kwargs):
+        if len(kwargs.keys()) == 0:
+            self._owns_data = 0
+            return
+        
+        self._owns_data = 1
```

Comment:
Under what situation is ``_owns_data`` ever 0? Think of why this was used in Target/TargetArray.